.github/workflows/expected-release-version.yml

```diff
@@ -0,0 +1,102 @@
+# add comment on PR for expected release version based on the commit messages
+
+name: Expected Release Version
+on:
+  - workflow_call
+
+jobs:
+  release-type:
+    name: Check Release Type
+    runs-on: ubuntu-latest
+    outputs:
+      release_level: ${{ steps.release_type.outputs.RELEASE_TYPE }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ env.GITHUB_SHA }}
+      - name: Get Commit Messages
+        id: get_commit_messages
+        run: |
+          PR_NUMBER=${{ github.event.pull_request.number }}
+          COMMITS=$(curl -s -H "Accept: application/vnd.github.v3+json" \
+                          -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
+                          "https://api.github.com/repos/${{ github.repository }}/pulls/$PR_NUMBER/commits")
+          COMBINED_MESSAGES=$(echo "$COMMITS" | jq -r '.[].commit.message' | paste -sd ' ')
+          echo "COMBINED_MESSAGES=$COMBINED_MESSAGES" >> $GITHUB_OUTPUT
+      - name: Get Release Type
+        id: release_type
+        run: |
+          if echo "${{ steps.get_commit_messages.outputs.COMBINED_MESSAGES }}" | grep -q "BREAKING CHANGE"; then
```

Comment:
I understand the purpose behind identifying `feat` and `fix` but why use `breaking change`?
Are we making the assumption that the phrase `Breaking Change` will already be present in the commit message?

Reply:
As per my understanding, according to angular conventional commits and semantic release docs, for major version to be released, there would be BREAKING CHANGE in commit message.